Assets/Scripts/PositionTracker.cs

```diff
@@ -6,16 +6,15 @@ public class PositionTracker : MonoBehaviour
 {
     private int currPathIndex = 0;
     private int currPosition = -1;
-
     private bool hasFinishedRace;
     private ScoreData score;
-    private float StartTime;
+    private ScoreManager scoreManger;
     private void Start()
     {
         score = new ScoreData(gameObject.name);
         hasFinishedRace = false;
         RacerManager.AddTracker(this);
-        StartTime = Time.time;
+        scoreManger = GameObject.Find("Race Track").GetComponent<ScoreManager>();
```

Comment:
Never use GameObject.Find given how slow it is. The ScoreManager is a singleton and therefore you do not need to have a ScoreManager instance to interact with it. 

If you created new functions for the ScoreManager they should be static so that they can be called from anywhere in the code base